common/components/multi-file-upload/multi-file-upload.js

```diff
@@ -116,7 +116,24 @@ MultiFileUpload.prototype = {
           row.parentNode.removeChild(row)
           this.render()
         })
-        .catch(() => false)
+        .catch(error => {
+          let errorMessage = 'could not be deleted'
+          if (error.response && error.response.data) {
+            errorMessage = error.response.data
+          }
+          const row = target.closest('.dz-success')
+          row.classList.remove('dz-success')
+          row.classList.add('dz-error')
+          row.querySelector('[data-dz-errormessage]').innerHTML = errorMessage
+          const buttons = document.querySelectorAll(
+            '.govuk-summary-list__actions button'
```

Comment:
Wonder if there's a better selector than GOV.UK Design System classnames. 

It's nice to not tie JS modules to classes used for presentation. If there's need for hooks we could use JS prefix classes `js-MultiFileUpload-hookName`?

Reply:
How about 

```
          const buttons = row
            .closest('.app-multi-file-upload__list')
            .querySelectorAll('button[name="delete"]')
```